src/forms/FormFieldAdvancedSelection/AdvancedSelectionDrawer.tsx

```diff
@@ -261,6 +269,9 @@ const AdvancedSelectionDrawer = (props: AdvanceSelectionDrawerPropTypes): ReactE
 	}, [fieldDef?.disabled, options]);
 
 	const deleteSelectedOption = (newOptions: MosaicLabelValue[]) => {
+		if (newOptions.length <= fieldDef.inputSettings?.selectLimit) {
+			setCheckboxListDisabled(false)

```

Comment:
Please don't forget the ";" at the end of each line. It might not be as important in javascript but in typescript it can lead to production errors.